scripts/zones/Ifrits_Cauldron/mobs/Magma.lua

```diff
@@ -3,24 +3,17 @@
 --  MOB: Magma
 -----------------------------------
 
------------------------------------
--- onMobSpawn Action
------------------------------------
-
 function onMobSpawn(mob)
+    DespawnMob(mob:getID(), 180);
```

Comment:
whats this intended for?

Reply:
sidenote: kinda bugs me that this one has the onMobInit after instead of before like the others, even though I know it doesn't effect anything really.

Reply:
c&p mistake :)

Reply:
so, this DespawnMob still exists in the most resent version
was that intended and you just meant the function order was copy paste error?

Reply:
it is itended for salamander, as he doesnt need to be defeated. makes no sense on magma as he spawns claimed by the trading player

Reply:
ah ok,